src/Features/Core/Portable/Completion/CompletionService.cs

```diff
@@ -169,5 +170,45 @@ public virtual TextSpan GetDefaultCompletionListSpan(SourceText text, int caretP
 
             return bestItems.ToImmutableAndFree();
         }
+
+        internal static async Task<CompletionDescription> GetDescriptionAsync(
+            CompletionService service, CompletionItem item, CancellationToken cancellationToken = default)
```

Comment:
So the problem as an instance method is that we already have the instance method.  and we'll conflict.